app/views/categories/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="w-full">
+  <% if notice.present? %>
+    <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
+  <% end %>
+
+  <div class="flex justify-between items-center">
+    <h1 class="font-bold text-white text-4xl absolute top-5 left-0 text-center w-screen m-0">Categories</h1>
+  </div>
+
+  <div id="categories" class="min-w-full">
+    <%= render @categories %>
+  </div>
+  <%= link_to 'Add new category', new_category_path, class: "rounded-lg py-3 px-5 bg-blue-600 text-white text-center block font-medium" %>
```

Comment:
- [ ] [OPTIONAL] From the previous review, the `Add New Category` button was meant to be sticky to the button of the screen. Let's endeavor to improve upon this aspect.

![Uploading image.png…]()

Reply:
I will fix it